### PR TITLE
Cleiton

### PR DESCRIPTION
**Pull request by frcleiton**
html-exercise 1.i and 1.ii

### CODE REVIEW

File: cleiton-ferreira/1.i/index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        <li>José Luiz da Silva</li>
+        <li>Júlio César Pereira</li>
+        <li>Kleber da Silva Garcia</li>
+        <li>Márcio Emílio Cruz Vono de Azevedo</li>
+        <li>Paula Chiaretti</li>
+        <li>Roberto Ribeiro Rocha</li>
+        <li>Valéria Santos Paduan Silva</li>
+    </ul>
+    <h2>Informação</h2>
+        <p>Para mais informações ou dúvidas entre em contato.</p>
```
Comment (edysegura) [inline cleiton-ferreira/1.i/index.html]: @frcleiton This paragraph is with misaligned indentation, don't you think?

File: cleiton-ferreira/1.i/index.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Sistemas de Informação Home Page</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (edysegura) [inline cleiton-ferreira/1.i/index.html]: Why did you use this tag? Are you preparing it for mobile device as well?

**Review by edysegura — COMMENTED** [review]
Hi @frcleiton! 

Very well done! I have only few question for you!

File: cleiton-ferreira/1.i/index.html (hunk 2 above)
Comment (frcleiton) [reply cleiton-ferreira/1.i/index.html]: Hi Edy, yes.

File: cleiton-ferreira/1.i/index.html (hunk 1 above)
Comment (frcleiton) [reply cleiton-ferreira/1.i/index.html]: I agree

File: cleiton-ferreira/1.ii/index.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (edysegura) [inline cleiton-ferreira/1.ii/index.html]: The content of this page is in English or in Portuguese?? How about change it to `lang="pt-br"`

File: cleiton-ferreira/1.i/index.html [hunk 4]
```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (edysegura) [inline cleiton-ferreira/1.i/index.html]: The content of this page is in English or in Portuguese?? How about change it to `lang="pt-br"`

File: cleiton-ferreira/1.ii/index.html [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+<head>
+    <meta charset="UTF-8">
+    <title>Curriculum Vitae</title>
+    <link rel="stylesheet" href="https://www.w3schools.com/lib/w3.css">
+</head>
+<body>
+    <table align="center">
```
Comment (edysegura) [inline cleiton-ferreira/1.ii/index.html]: Please don't use this attribute anymore `align="center"` it breaks the meaning of HTML. You can do the same with CSS.

File: cleiton-ferreira/1.ii/index.html [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+        <tr>    
+            <th><img src="cleiton.jpg" alt="Cleiton" title="Foto" width="150" height="150" /></th>
```
Comment (edysegura) [inline cleiton-ferreira/1.ii/index.html]: Avoid to use these attributes `width="150" height="150"`

File: cleiton-ferreira/1.ii/index.html [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+<head>
+    <meta charset="UTF-8">
+    <title>Curriculum Vitae</title>
+    <link rel="stylesheet" href="https://www.w3schools.com/lib/w3.css">
+</head>
+<body>
+    <table align="center">
+        <tr>    
+            <th><img src="cleiton.jpg" alt="Cleiton" title="Foto" width="150" height="150" /></th>    
+            <th align="right"><h2>Curriculum Vitae</h2></th>
```
Comment (edysegura) [inline cleiton-ferreira/1.ii/index.html]: Same here `align="right"`

File: cleiton-ferreira/1.ii/index.html [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+</head>
+<body>
+    <table align="center">
+        <tr>    
+            <th><img src="cleiton.jpg" alt="Cleiton" title="Foto" width="150" height="150" /></th>    
+            <th align="right"><h2>Curriculum Vitae</h2></th>    
+        </tr>
+        <tr>
+            <th colspan="2">
+                <p><h3>Cleiton Rezende Ferreira<h3></p>
```
Comment (edysegura) [inline cleiton-ferreira/1.ii/index.html]: It is not right to use `<h3>` inside a `<p>`

File: cleiton-ferreira/1.ii/index.html [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+                <p>35 anos, casado, 2 filhos</p>
+                <p>Rua Maria Guilhermina Franco, 67</p>
+                <p>Pouso Alegre - MG 37550-000</p>
+                <p>(35) 99118-8928</p>
+            </th>
+        </tr>
+    </table>
+    <h3><u>Formação Acadêmica</u></h3>
+    Bacharelado em Sistemas de Informação – Universidade do Vale do Sapucaí – Pouso Alegre - MG
+    <p></p>
```
Comment (edysegura) [inline cleiton-ferreira/1.ii/index.html]: Why this empty `<p>`

Comment (edysegura) [reply cleiton-ferreira/1.ii/index.html]: There are more empty `<p>` below!

**Review by edysegura — CHANGES_REQUESTED** [review]
Hi @frcleiton!

Please take a look in my comments and let me know if you have doubts.

Cheers

File: cleiton-ferreira/1.ii/index.html (hunk 3 above)
Comment (frcleiton) [reply cleiton-ferreira/1.ii/index.html]: Done

File: cleiton-ferreira/1.ii/index.html (hunk 5 above)
Comment (frcleiton) [reply cleiton-ferreira/1.ii/index.html]: done

File: cleiton-ferreira/1.i/index.html (hunk 4 above)
Comment (frcleiton) [reply cleiton-ferreira/1.i/index.html]: Hi Teacher, thanks for your comments and support.

**Comment by frcleiton** [comment]
The index.html has been fixed.

**Review by edysegura — COMMENTED** [review]
Good job!! 👍

**Review by edysegura — APPROVED** [review]

**Pull request merged**